alembic_migration/postgresql/env.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from pathlib import Path
+
+import alembic_postgresql_enum  # noqa: F401
+from alembic import context
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+
+from tdp.core.models.base_model import BaseModel
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Model's MetaData object here for 'autogenerate' support.
+target_metadata = BaseModel.metadata
+
+
+def run_migrations_offline() -> None:
+    """Run migrations in 'offline' mode.
+
+    This configures the context with just a URL
+    and not an Engine, though an Engine is acceptable
+    here as well.  By skipping the Engine creation
+    we don't even need a DBAPI to be available.
+
+    Calls to context.execute() here emit the given string to the
+    script output.
```

Comment:
We should move those docs to the `README`, along with the quick explanation of the online/offline modes. It'll help avoiding to maintain 3 versions of it.